server/handler.go

```diff
@@ -397,3 +400,71 @@ func (h *Handler) AddRemovePeerOperator(regionID uint64, fromStoreID uint64) err
 	c.addOperator(op)
 	return nil
 }
+
+// ResponseDownPeerRegions gets the region with down peer.
+func (h *Handler) ResponseDownPeerRegions(w http.ResponseWriter, rd *render.Render) {
+	c := h.s.GetRaftCluster()
+	if c == nil {
+		rd.JSON(w, http.StatusInternalServerError, errNotBootstrapped)
+		return
+	}
+	cluster := c.cachedCluster
+	if cluster.regionStats == nil {
+		rd.JSON(w, http.StatusInternalServerError, errRegionStatsNotFound)
+		return
+	}
+	cluster.RLock()
+	rd.JSON(w, http.StatusOK, cluster.regionStats.stats[downPeer])
```

Comment:
I prefer add function getXXX() in clusterInfo and manage lock by clusterInfo itself.